docs/src/ttmlir-translate.md

```diff
@@ -0,0 +1,12 @@
+# `ttmlir-translate`
+
+The `ttmlir-translate` translation utility. Unlike `ttmlir-opt` tool which is used to run passes within the MLIR world, `ttmlir-translate` allows us to ingest something (e.g. code) into MLIR world, and also produce something (e.g. executable binary, or even code again) from MLIR.
+## Generate C++ code from MLIR
+
+```bash
+# First, let's run `ttmlir-opt` to convert to proper dialect
+./build/bin/ttmlir-opt --ttir-layout --ttnn-open-device --convert-ttir-to-ttnn --convert-ttnn-to-emitc test/ttmlir/Dialect/TTNN/simple_multiply.mlir -o c.mlir
```

Comment:
Just out of curiosity, can these commands be piped?

Reply:
I scoured thru all the options for the tool and it seemed it couldn't be done... I tried it now, and it works lol

```bash
./build/bin/ttmlir-opt --ttir-layout --ttnn-open-device --convert-ttir-to-ttnn --convert-ttnn-to-emitc test/ttmlir/Dialect/TTNN/simple_multiply.mlir | ./build/bin/ttmlir-translate -mlir-to-cpp -allow-unregistered-dialect
```

Reply:
Wow! Great, consider adding that example as well, it skips creating the unnecessary file on the file system.